modules/post/linux/gather/phpmyadmin_credsteal.rb

```diff
@@ -0,0 +1,67 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Post
+
+  include Msf::Post::File
+  include Msf::Post::Linux::Priv
+  include Msf::Post::Linux::System
+
+  def initialize(info={})
+    super(update_info(info,
+      'Name'         => "PhpMyAdmin credentials stealer",
+      'Description'  => %q{
+        This module gathers PhpMyAdmin Creds from Target Linux machine.
+      },
+      'License'      => MSF_LICENSE,
+      'Platform'     => ['linux'],
+      'SessionTypes' => ['meterpreter'],
+      'Arch'         => 'x86_x64',
```

Comment:
This can removed. Architecture is almost certainly redundant for a module that wraps `read_file`.